### ISSUE TITLE
Vote players off

### ISSUE THREAD

**Issue by popey** [grifferz/pah-irc#15]
Sometimes a player will go afk and cause the game to be stuck. Both the tsar and other players can lock the game up until they return.

It should be possible for all the remaining players to have a vote.

Perhaps one person initiates the vote and once a %age of those +1 (vs -1/0) it, it passes and the afk player is removed.

**Comment by grifferz** [comment]
I don't think that this is particularly necessary given a `turnclock` that everyone agrees with.

For example, the turnclock in `#bitfolk` is one day, so there is little justification for voting someone off until they are forcibly resigned anyway.

I could see the point of it in the case where everyone knows that the user will not be back, so they are faced with a pointless delay of a day. Yet if the game does not progress for a day then it is hardly a big blow to life in `#bitfolk`.

It probably makes slightly more sense for faster-paced games such as the one in `#debug` which has a one hour turnclock. If everyone knows that a user will not be back then an enforced delay of one hour could be more demoralising for a game that had previously been kicking along quite nicely.

So, I'm not going to close this as `wontfix`, but I probably won't work on implementing this for a while (if ever) as I'm not too interested in supporting fast-paced games. I am not opposed to accepting a pull request that implements it in the mean time.

A general voting infrastructure will likely be necessary for other feature requests, so that part is what should happen first.
